Makefile

```diff
@@ -5,7 +5,7 @@ SHELL = bash
 RHOAS_VERSION ?= "dev"
 REPOSITORY_OWNER ?= "redhat-developer"
 REPOSITORY_NAME ?= "app-services-cli"
-TERMS_SPEC_URL ?= "https://raw.githubusercontent.com/redhat-developer/app-services-ui/main/static/configs/terms-conditions-spec.json"
+TERMS_SPEC_URL ?= "https://console.redhat.com/apps/application-services/terms-conditions-spec.json"
```

Comment:
We will probably want a different one for stage and prod?

Reply:
Not sure why we would need it?

Reply:
Spec file could be different between stage and prod versions

Reply:
We do not have ability to detect env and we need more sophisticated remote config for massso. I think this requirement would need more design etc. 

If we need config agains stage I think we should build different version of cli for noe

Reply:
I think this tc config should be refactored to more generic config. Created issue to cover this work but this is out of scope of current PR